app/src/main/java/org/wikipedia/bridge/JavaScriptActionHandler.kt

```diff
@@ -38,6 +38,12 @@ object JavaScriptActionHandler {
         return "window.scrollTo(0, document.getElementById('pcs-footer-container-menu').offsetTop - ${DimenUtil.getNavigationBarHeight(context)});"
     }
 
+    @JvmStatic
+    fun scrollToAnchor(anchorLink: String): String {
+        val anchor = if (anchorLink.contains("#")) anchorLink.substring(anchorLink.indexOf("#") + 1) else anchorLink
+        return "var el = document.getElementById('$anchor'); window.scrollTo(0, el.offsetTop - (screen.height / 2)); setTimeout(function(){ el.style.backgroundColor='#ee0'; setTimeout(function(){ el.style.backgroundColor=null; }, 500); }, 250);"
```

Comment:
Would it be better if it has multiple lines to increase the readability?

Reply:
ok!